src/Compiler/RefC/RefC.idr

```diff
@@ -26,8 +26,6 @@ import System.File
 import Protocol.Hex
 import Libraries.Utils.Path
 
-%default covering
```

Comment:
This is a pretty big change. Are you sure we don't want any functions checked (that weren't already marked partial) in this module?

Reply:
This is a wrong commit. I have revoked it.